README.md

```diff
@@ -154,3 +155,48 @@ i.e. for sandbox
 ```bash
 $ az identity show --name cnp-sandbox-mi -g managed-identities-sbox-rg --subscription DCD-CFT-Sandbox --query principalId -o tsv
 ```
+### Workload Identity
+A Federated Identity Credential will be created with the name `$product-$env-fdc` for using Workload Identity on AKS. The subject of this resource is `system:serviceaccount:${local.namespace}:${local.namespace}` - ideally you will supply `namespace` as a variable when calling this module:
+
+```hcl
+module "this" {
+  source              = "git@github.com:hmcts/cnp-module-key-vault?ref=master"
+  name                = "rhubarb-fe-${var.env}" // Max 24 characters
+  product             = var.product
+  namespace           = "Your team's AKS namespace" # e.g. sscs, xui, dm-store
+  env                 = var.env
+  object_id           = var.jenkins_AAD_objectId
+  resource_group_name = azurerm_resource_group.rg.name
+  product_group_name  = "Your AAD group" # e.g. MI Data Platform, or dcd_cmc
```

Comment:
no need to duplicate all args each time it makes it harder to see what is relevant in the actual example
```suggestion
  ...
```